lib/itamae/generators/templates/project/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'itame'
+gem 'serverspec'
```

Comment:
Because Serverspec is not required by Itamae, IMO it is better to be commented out.